src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2025,7 +2024,7 @@ namespace RCT1
             }
             else
             {
-                ResearchItem researchItem = {};
+                ::ResearchItem researchItem = {};
```

Comment:
(More a note for a future PR)
To avoid global `::...`, we should move these in an OpenRCT2 namespace and change these types to `OpenRCT2::...`.